<?xml version="1.0" encoding="UTF-8"?>
<!--
# Program Name:              vis
# Program Description:       Measures sequences of vertical intervals.
#
# Filename: vis/views/ui/select_offset.ui
# Purpose: UI file for the "select offset" component.
#
# Attribution:  Based on the 'harrisonHarmony.py' module available at...
#               https://github.com/crantila/harrisonHarmony/
#
# Copyright (C) 2012, 2013 Christopher Antila, Jamie Klassen
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->

<ui version="4.0">
 <class>Select_Offset</class>
 <widget class="QDialog" name="Select_Offset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Choose an Offset Duration</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>music21 Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_music21_duration">
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0.5</string>
        </property>
        <property name="maxLength">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QToolButton" name="btn_0_25">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/sixteenth.png</normaloff>:/icons/icons/sixteenth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btn_2">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/half.png</normaloff>:/icons/icons/half.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="btn_1">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/quarter.png</normaloff>:/icons/icons/quarter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="btn_4">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/whole.png</normaloff>:/icons/icons/whole.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="btn_8">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/breve.png</normaloff>:/icons/icons/breve.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="btn_0_5">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/eighth.png</normaloff>:/icons/icons/eighth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="btn_0_125">
        <property name="text">
         <string>32nd</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/thirty_second.png</normaloff>:/icons/icons/thirty_second.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="btn_0_0625">
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/sixty_fourth.png</normaloff>:/icons/icons/sixty_fourth.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="btn_all">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>all</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>332</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_submit">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/show_checkmark.png</normaloff>:/icons/icons/show_checkmark.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
